submissions/drewzag/tjsw-oop/index.js

```diff
@@ -0,0 +1,67 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/drewzag/a-tiny-JS-world
+   Web app: https://drewzag.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Inhabitant {
+  constructor(species, name, gender, legs, saying) {
+    this.species = species
+    this.name = name
+    this.gender = gender
+    this.legs = legs
+    this.saying = saying
+  }
+
+  printInhabitants(args) {
```

Comment:
`args` is a generic term. It doesn't tell what the method expects.
When, as a peer developer, I type `classIntanceVariable.printInhabitants(` the IDE intellisence hints that next thing to type is some "args". There is no way for me to know what the function expects to receive without browsing the code and reading it. Meaningful names help to resolve this at function usage time.

I have an idea of what you want to do here. Note that every class should handle its own properties.
Use method overloading/overriding instead.
OOP module theory covered this. Consider revisiting